vowpalwabbit/cbify.cc

```diff
@@ -145,8 +157,7 @@ void copy_example_to_adf(cbify& data, example& ec)
     // offset indices for given action
     for (features& fs : eca)
     {
-      for (feature_index& idx : fs.indicies)
-      { idx = ((((idx >> ss) * 28904713) + 4832917 * (uint64_t)a) << ss) & mask; }
+      for (feature_index& idx : fs.indicies) { idx = (((idx >> ss) * num_actions + a) << ss) & mask; }
```

Comment:
do we know what these constants were?

Reply:
Not sure how those constants were concocted in the past, but this change is to avoid collision between the actions on the same example. @pmineiro feel free to chime in this one too.

Reply:
Are we sure it is okay to remove these constants? This would break existing users of cbify no?

Reply:
cbify is only used for research, and the details of how it creates contextual bandit example features are not in any public contract.